x-pack/test/detection_engine_api_integration/security_and_spaces/group10/update_rules_bulk.ts

```diff
@@ -45,6 +45,7 @@ export default ({ getService }: FtrProviderContext) => {
   const supertest = getService('supertest');
   const log = getService('log');
   const es = getService('es');
+  const esArchiver = getService('esArchiver');
 
```

Comment:
can we add test, that not migrated rule continues to execute?

This can happen if rule has investigation fields and is enabled, but not edited by user

Reply:
Manually confirmed rules are now running with updates to code. Thanks for catching this!